bin/generate-changelog.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
```

Comment:
Will this script work on all platforms ? If not, platform requirements to run the script should probably be mentioned in footnote 2 and as documentation at the top of this file.

Reply:
I runs on Bash, so it'll work on any platform that runs bash. I'd hope that `#!/usr/bin/env bash` is clear enough, from inside the script.

I'll add mention at the footnote, though. :+1: